packages/unit-testing/stubs/action/action.js

```diff
@@ -0,0 +1,127 @@
+     * @param {string=} options.id action ID
+     * @return {Object} a set of actions (@see Action) defined on the record type indexed by action ID
+     * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if options.recordType is missing or undefined
+     * @throws {SuiteScriptError} SSS_INVALID_RECORD_TYPE if the specified record type doesn't exist
+     * @throws {SuiteScriptError} SSS_INVALID_ACTION_ID if an action is specified and such action doesn't exist on the said record type
+     * @throws {SuiteScriptError} CONDITION_IS_NOT_FULLY_QUALIFIED if the action does not qualify on the specific record instance
+     * @throws {SuiteScriptError} RECORD_DOES_NOT_EXIST if a record ID is specified and that record instance doesn't exist
+     *
+     * @since 2018.2
+     */
+    action.prototype.find = function (options) { };
+    action.prototype.find.promise = function (options) { };
+
+    /**
+     * Returns an executable record action for the given record type. If the recordId parameter is provided, then the
+     * action object is only returned if the given record instance qualifies for execution of the given record action.
+     * Also, if recordId is provided than the returned action is "qualified" and you don't have to provide the recordId
+     * again when executing the Action object.
+     *
+     * @governance none
+     *
+     * @param {Object} options
+     * @param {string} options.recordType record type
+     * @param {string} options.id action ID
+     * @param {string=} options.recordId record instance ID
+     * @return {Action} record action executor for action specified by options
+     * @throws {SuiteScriptError} SSS_MISSING_REQD_ARGUMENT if options.recordType or options.id is missing or undefined
+     * @throws {SuiteScriptError} SSS_INVALID_RECORD_TYPE if the specified record type doesn't exist
+     * @throws {SuiteScriptError} SSS_INVALID_ACTION_ID if the specified action doesn’t exist on the said record type
+     * @throws {SuiteScriptError} CONDITION_IS_NOT_FULLY_QUALIFIED if the action does not qualify on the specific record instance
+     * @throws {SuiteScriptError} RECORD_DOES_NOT_EXIST if a record ID is specified and that record instance doesn't exist
+     *
+     * @since 2018.2
+     */
+    action.prototype['get'] = function (options) { };
+    action.prototype['get'].promise = function (options) { };
+
+    /**
+     * Executes the record action and returns the action results in a plain JavaScript object.
+     * If the action fails, it is listed in the results object’s notifications property. If the action executes successfully, the notifications property is usually empty.
+     *
+     * @governance 50 units for timesheets, 10 for transactions, 2 for custom records, 5 for all other records
+     *
+     * @param {Object} options
+     * @param {string} options.recordType record type
+     * @param {string} options.id action ID
+     * @param {Params} options.params action arguments
```

Comment:
I don't know if its going to much into detail but I think this function is missing a parameter

add: '* @param {string} options.params.recordId record instance ID'

According to https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=section_1509391388.html

Also governance there seems to be none but I guess is not important